app/queries/consumer_apps/find_or_create_all_query.rb

```diff
@@ -5,7 +5,15 @@ def self.call
       existing_platforms = ConsumerApp.where(app_bundle: forem_bundle).pluck(:platform)
       (ConsumerApp::FOREM_APP_PLATFORMS - existing_platforms).each do |platform|
         # Re-create the supported Forem apps if they're missing
-        ConsumerApp.create(app_bundle: forem_bundle, platform: platform, active: true)
+        ConsumerApp.create!(app_bundle: forem_bundle, platform: platform, active: true,
```

Comment:
`active` defaults to `true` in the schema, so maybe we don't need to explicitly list it here?

```ruby
  create_table "consumer_apps", force: :cascade do |t|
    t.boolean "active", default: true, null: false
```